checks.d/daemontools.py

```diff
@@ -0,0 +1,39 @@
+# project
+from checks import AgentCheck
+import re
+import subprocess
+
+dt_status = re.compile(".*: (down|up) \(pid (\d+)\) (\d+) seconds")
+
+class DaemonToolsCheck(AgentCheck):
+    def check(self, instance):
+        path = self.init_config.get('path', '/etc/service')
+        service = instance.get('service')
+        tags = instance.get('tags', [])
+        tags.append("service:" + service)
+
+        try:
+            status = subprocess.Popen(['svstat', path + "/" + service], stdout=subprocess.PIPE, close_fds=True).communicate()[0]
```

Comment:
You should use https://github.com/DataDog/dd-agent/blob/master/utils/subprocess_output.py#L15 instead. It's safer to use.